files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,442 @@ class AaaCfg(object):
+                if type(it) is unicode:
+                    key = it
+                else:
+                    key = it[0] + "|" + it[1]
+
+                entry = self.config_db.get_entry(CFG_MGMT_INTF_TABLE, key)
+                if 'gwaddr' in entry:
+                    mgmt_intf_gwaddr = entry['gwaddr']
+                    cmd = 'ip route add table {} 0.0.0.0/0 via {}'.format(MGMT_VRF_TABLE_ID, mgmt_intf_gwaddr)
+                    if run_command(cmd) == False:
+                        return
+
+    def unconfigure_mgmt_vrf(self):
+        syslog.syslog(syslog.LOG_INFO, "Unconfigure MGMT VRF:")
+
+        cmd = 'ip link delete lo-m'
+        if run_command(cmd) == True:
+            self.delete_mgmt_vrf_table()
+
+        return
+
+    def handle_mgmt_intf_gwaddr_cfg(self, op, ifname, gwaddr):
+        syslog.syslog(syslog.LOG_INFO, "Received MGMT Intf gwaddr config, op: {}, ifname: {}, gwaddr: {}".format(op, ifname, gwaddr))
+        keys = self.config_db.get_keys(CFG_MGMT_VRF)
+        if keys:
+            cmd = 'ip route del table {} 0.0.0.0/0'.format(MGMT_VRF_TABLE_ID)
+            run_command(cmd)
+            
+            if op == "ADD":
+                cmd = 'ip route add table {} 0.0.0.0/0 via {}'.format(MGMT_VRF_TABLE_ID, gwaddr)
+                run_command(cmd)
+            return True
+
+    def handle_mgmt_intf_creation(self, mgmt_intf_key):
+        syslog.syslog(syslog.LOG_INFO, "Received MGMT Intf creation, key: {}".format(mgmt_intf_key))
+        keys = self.config_db.get_keys(CFG_MGMT_VRF)
+        if keys:
+            syslog.syslog(syslog.LOG_INFO, "MGMT VRF created")
+            self.bind_mgmt_intf()
+
+    def handle_mgmt_vrf_cfg(self, key, data):
+        syslog.syslog(syslog.LOG_INFO, "Received MGMT VRF config, key: {}, data: {}".format(key, data))
+        try:
+            if data:
+                if data['mgmtVrfEnabled'] == 'true':
+                    self.configure_mgmt_vrf()
+                else:
+                    self.unconfigure_mgmt_vrf()
+            else:
+                self.unconfigure_mgmt_vrf()
+        except:
+            syslog.syslog(syslog.LOG_ERR, "{} MGMT VRF Configuration failed.".format(key))
+
+    def mgmt_vrf_table_get(self):
+        vrf_table = 'default'
+        entry = self.config_db.get_entry(CFG_MGMT_VRF, CFG_MGMT_VRF_KEY)
+        if entry:
+            if (('mgmtVrfEnabled' in entry) and (entry['mgmtVrfEnabled'] == 'true')):
+                vrf_table = '5000'
+
+        return vrf_table
+
+
+
+
+def run_command(cmd):
+    syslog.syslog(syslog.LOG_INFO, "Config cmd - {}".format(cmd))
+    try:
+        subprocess.check_call(cmd, shell=True)
+    except subprocess.CalledProcessError as err:
+        syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
+                .format(err.cmd, err.returncode, err.output))
+        return False
+
+    return True
+
+def cfg_attr_set (cmd, attr = None, value = None, appIntf = None):
+
```

Comment:
Most of your functions doesn't have empty line at start. Please remove this line to be consistent

Reply:
done